### PR TITLE
Tweak travis

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

**Comment by lsmith77** [comment]
@xabbuh / @Ener-Getick good to merge?

scrutinzer seems to not have received any info to build .. but lets not worry about that

**Comment by Ener-Getick** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ language: php
 
 php:
   - 5.4
+  - 5.5
   - 5.6
   - 7.0
```
Comment (xabbuh) [inline .travis.yml]: We can remove this line to save one job.

Comment (lsmith77) [inline .travis.yml]: but don't we then miss running php 7.0 with symfony 3.0 ?

Comment (xabbuh) [inline .travis.yml]: We would. But is that necessary? I mean we already run Symfony 3.0 tests with other PHP versions.

Comment (lsmith77) [inline .travis.yml]: imho its cleaner to do this ..

**Pull request merged**